docs/SQLModelingStandard.md

```diff
@@ -26,7 +26,7 @@ infrastructure will be re-designed to follow this guideline.*
   of the words in the name should reveal the concern of the table in a 
   broad-to-specific manner. 
   - *i.e. A table containing the consent information of a candidate would be 
-    named `candidate_consent`, candidate being the broader concern and just 
+    named `candidate_consent`, candidate being the broader concern and 
```

Comment:
not the same `just` as in `simply` -- this one means `only`.  
if it is to be removed, it should be replaced by a comma to retain the clarity that it contributed to the sentence structure

Reply:
@christinerogers I reworded the sentence. let me know if that works for you

Reply:
sure, and thanks